_includes/footer.html

```diff
@@ -5,6 +5,11 @@
 
       <footer>
         <ul class="list-inline">
+          {% if page.path contains 'docs' %}
```

Comment:
I'm not sure this check is reliable. For example, the edit button at https://j-f1.github.io/eslint.github.io/docs/developer-guide/contributing/ goes to a 404 because it links to `index.md` rather than `README.md`.

It also feels a bit fragile to do a substring check -- for example, if we added a page to `eslint.github.io` called `how-to-write-docs.md`, it would end up linking to the `eslint` repo.